messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/jms/JMSTopicControlHandler.java

```diff
@@ -388,6 +407,30 @@ private String listSubscribersInfosAsJSON(final DurabilityType durability, Addre
       }
    }
 
+   private void pause(AddressControl control, boolean persist) {
+       try {
+       control.pause(persist);
+       } catch (Exception e) {
+         throw new RuntimeException(e);
```

Comment:
Perhaps add a separate catch for RuntimeException and just rethrow it instead of wrapping?

Same comment applies to resume and isPaused.

Reply:
There is actually a way to rethrow both without wrapping in a RuntimeException - see https://github.com/resteasy/Resteasy/blob/43f46228bbf8278a96648b48309f71d5ccad1e4c/resteasy-core/src/main/java/org/jboss/resteasy/core/SynchronousDispatcher.java#L186

Reply:
ok, using Kabir's trick. This should be applied to the rest of the code though in a separate commit